robottelo/performance/perf_thread.py

```diff
@@ -0,0 +1,92 @@
+"""Test utilities for multi-threading programming"""
+from robottelo.performance.perf_candlepin import CandlepinPerfTest
+
+import logging
+import threading
+import time
+
+
+class PerfThread(threading.Thread):
```

Comment:
s/PerfThread/PerformanceThread from the zen of python, explicit is better than implicit. Perf could mean other thing for other person.